docs/config/README.md

```diff
@@ -46,9 +46,10 @@ This is the list of prompt-wide configuration options.
 
 ### Options
 
-| Variable      | Default | Description                                    |
-| ------------- | ------- | ---------------------------------------------- |
-| `add_newline` | `true`  | Add a new line before the start of the prompt. |
+| Variable       | Default | Description                                                        |
+| -------------- | ------- | ------------------------------------------------------------------ |
+| `add_newline`  | `true`  | Add a new line before the start of the prompt.                     |
+| `prompt_order` | `[]`    | Configure out custom prompt order overriding [default prompt order](#default-prompt-order). |
```

Comment:
I think this may be a slightly clearer way of explaining this option.

```suggestion
| `prompt_order` | [link](#default-prompt-order)    | Configure the order in which the prompt modules appear. |
```